### PR TITLE
Fix target_versions for FileMode

### LINKED ISSUE #141: Error with black==22.1.0
My pre-commit config:
```yaml
  - repo: https://github.com/asottile/blacken-docs
    rev: v1.12.0
    hooks:
      - id: blacken-docs
        additional_dependencies: [ black==22.1.0 ]
```

Error:
```
blacken-docs.............................................................Failed
- hook id: blacken-docs
- exit code: 1

Traceback (most recent call last):
  File "/tmp/ppfeufer/.cache/pre-commit/repoh7brsv91/py_env-python3/bin/blacken-docs", line 8, in <module>
    sys.exit(main())
  File "/tmp/ppfeufer/.cache/pre-commit/repoh7brsv91/py_env-python3/lib/python3.9/site-packages/blacken_docs.py", line 238, in main
    black_mode = black.FileMode(
  File "<string>", line 3, in __init__
TypeError: set object expected; got list
```

Works perfectly fine with ``black==21.12b0`

pip show black:
```
Name: black
Version: 22.1.0
Summary: The uncompromising code formatter.
Home-page: https://github.com/psf/black
Author: Łukasz Langa
Author-email: lukasz@langa.pl
License: MIT
Location: /mnt/sda1/Development/Python/AllianceAuth/venv-3.9/lib/python3.9/site-packages
Requires: click, mypy-extensions, pathspec, platformdirs, tomli, typing-extensions
Required-by: 
```

### PR DESCRIPTION
**Pull request by JelleZijlstra**
Fixes #141

### CODE REVIEW

File: tests/blacken_docs_test.py [hunk 1]
```diff
@@ -414,14 +414,14 @@ def test_integration_multiple_target_version(tmpdir):
         'def very_very_long_function_name(\n'
         '    very_very_very_very_very_very,\n'
         '    very_very_very_very_very_very,\n'
-        '    *long_long_long_long_long_long\n'
+        '    *long_long_long_long_long_long,\n'
```
Comment (asottile) [inline tests/blacken_docs_test.py:417]: this no longer demonstrates the behaviour that the option is honored -- the test would "pass" even if no options were being forwarded along

**Review by asottile — COMMENTED** [review]
I'm still not convinced this is correct -- the change in types appears to be a mistake in black

File: tests/blacken_docs_test.py (hunk 1 above)
Comment (JelleZijlstra) [reply tests/blacken_docs_test.py:417]: That's true, I think I should pass an early-py3 target version instead.

Comment (JelleZijlstra) [reply tests/blacken_docs_test.py]: Fixed by using py35

**Pull request merged**